test/WebSites/BasicWebSite/Areas/Area1/Controllers/RemoteAttribute_HomeController.cs

```diff
@@ -1,41 +1,40 @@
-// Copyright (c) .NET Foundation. All rights reserved.
+﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using BasicWebSite.Models;
```

Comment:
Note GitHub is confused between the Area1 controller (previously the Aria controller) and the root controller. Comparing Area1 controller to old root controller.